package.json

```diff
@@ -63,12 +64,15 @@
     "dockerise": "cd docker && ./dockerise.sh",
     "docker_run": "cd docker && ./run.sh",
     "release": "./utils/release.sh",
-    "migrate": "ts-node ./database/migration/migrateData.ts --migrate"
+    "migrate": "ts-node ./database/migration/migrateData.ts --migrate",
+    "backup": "ts-node ./src/components/exportAndImport/backup.ts --backup"
   },
   "dependencies": {
     "@graphile-contrib/pg-simplify-inflector": "^6.0.0",
     "@openmsupply/expression-evaluator": "^1.11.2",
     "adm-zip": "^0.5.5",
+    "archiver": "^5.3.1",
+    "archiver-zip-encrypted": "^1.0.11",
```

Comment:
So there's two packages here -- "archiver" is a general purpose archiving library, and "archiver-zip-encrypted" is a plugin/extension for "archiver" that supports encryption.